docs/src/main/asciidoc/spring-cloud-config-client.adoc

```diff
@@ -0,0 +1,140 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Reading properties from Spring Config Server
```

Comment:
Should it be Spring Cloud Config Server?

Reply:
Sure, it sounds better